scripts/zones/Mount_Zhayolm/mobs/Cerberus.lua

```diff
@@ -37,5 +37,5 @@ end;
 -----------------------------------
 
 function onMobDespawn(mob)
-    mob:setRespawnTime((math.random(0,24)*3600)+172800); -- 48-72 hours proper 1 hour windows
+    mob:setRespawnTime(math.random(172800,259200)); -- 48-72 hours proper 1 hour windows
```

Comment:
Bleh, okay, I prefer consistence, and that gives me a better way to do the nm timers (adjust the window time, rather than the raw time), so I will work on that. Fortunately, work is dead at the moment xD